_config.yml

```diff
@@ -18,7 +18,7 @@
 # You can create any custom variable you would like, and they will be accessible
 # in the templates via {{ site.myvariable }}.
 
-title: Multipath TCP for Linux
+title: Multipath TCP<br>for Linux
```

Comment:
the HTML title is now `<title>Home | Multipath TCPfor Linux</title>`
also, not related but should you not use `<br />` instead?

Reply:
done